pep-0570.rst

```diff
@@ -37,15 +37,32 @@ of it. But, this is not always desired nor even available as
 even in current versions of Python, many CPython
 "builtin" functions still only accept positional-only arguments.
 
+Users might want to restrict their API to not allow for parameters
+to be referenced via keyword, as that exposes the name of the
+parameter as part of the API. If a user of said API starts using the
+argument by keyword when calling it and then the name of the parameter
+is changed, the code will be broken. By using positional-only parameters
```

Comment:
`the code will be broken` - > `the code will break` -> `it will be a breaking change`?